### ISSUE TITLE
Planned currencies

### ISSUE THREAD

**Issue by sagikazarmark** [umpirsky/currency-list#2]
Do you think it is possible to somehow add planned currencies to the list?

Reference: https://github.com/moneyphp/money/pull/193

**Comment by mohorev** [comment]
Yes. I think it is possible. E.g. now in Republic of Belarus prices are specified in both currencies concurrently BYR and BYN.

This is called the transition period. This is necessary so that the business can move smoothly to the new currency. Therefore, I would like to see the use of BYN code now.

**Comment by sagikazarmark** [comment]
I think it highly depends on the data source as these lists are not constructed by hand.

**Comment by mohorev** [comment]
Now I understand the question. I hope this will be possible. Maybe this official document will be useful. [Currency ISO amendment 161](http://www.currency-iso.org/dam/downloads/dl_currency_iso_amendment_161.docx)

**Comment by umpirsky** [comment]
Thanks for reporting guys. I submitted [WIP PR](https://github.com/umpirsky/currency-list/pull/3), looks like it is not still available. 

We depend on Symfony ICU component and it depends on ICU database, so we have this delay.

Once they update, it will be fairly easy to rebuild the list.

**Comment by mohorev** [comment]
@umpirsky These changes https://github.com/umpirsky/currency-list/pull/3 does not contains the `BYN` currency code. :cry:

**Comment by sagikazarmark** [comment]
@mongosoft yeah, as explained above. That's why I suggested the alternative in moneyphp/money#193

**Comment by umpirsky** [comment]
@mongosoft @sagikazarmark Yes, as I said we have this delay.

![guys-table-slo-q0FBs96PQmjDi](https://camo.githubusercontent.com/0705a3f3ca9b07d7b200f9a2d636d450aa099173/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7130464273393650516d6a44692f67697068792e676966)

**Comment by umpirsky** [comment]
I released `v1.1.0` with fresh currencies. :champagne:
